### ISSUE TITLE
Move over Contributors page to here

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble-AssemblyLine-documentation#166]
Now that this site serves as the main site for the project, we should copy over the contributors list from https://github.com/SuffolkLITLab/doc-assembly-line.

**Comment by BryceStevenWilley** [comment]
Started work on this in the [Contributors branch](https://github.com/SuffolkLITLab/docassemble-AssemblyLine-documentation/tree/contributors). Currently looks like the image below. We definitely want some minimal CSS to make it look better, but the existing CSS doesn't seem to play nice with docusaurus. 

Also, might be useful to think about integrating https://allcontributors.org/docs/en/specification, at least somewhere in our repositories.

![Screenshot from 2021-11-17 12-16-41](https://user-images.githubusercontent.com/6252212/142249477-8432d103-b887-4594-9267-324c4e857902.png)

**Comment by BryceStevenWilley** [comment]
More progress, I had to wrangle with JSX for quite a bit, but there's not a simple way to add new contributors without copy-pasting the entire HTML structures each time. I also made it look not bad using some tables, example below. I'm only about 1/4th of the way through the whole list, so I'll keep tackling bits when I can, but someone else should feel free to take this up if they want!
![Screenshot from 2021-12-01 00-15-52](https://user-images.githubusercontent.com/6252212/144176081-f4bbf6a0-a15b-4b43-b512-d017d189361d.png)
